packages/lodestar-validator/src/validator.ts

```diff
@@ -72,6 +72,7 @@ export class Validator {
   public async stop(): Promise<void> {
     this.isRunning = false;
     await this.apiClient.disconnect();
+    if (this.attestationService) await this.attestationService.stop();
```

Comment:
stop block service as well so if something gets added to block service stop method,  gets stopped as well

Reply:
done